ambry-server/src/test/java/com.github.ambry.server/AmbryRequestsTest.java

```diff
@@ -987,6 +987,9 @@ public long getSizeInBytes() {
         return 0;
       }
 
+      @Override
+      public boolean isEmpty() { return false; }
```

Comment:
You might want to follow these steps for adopting the ambry codestyle: https://github.com/linkedin/ambry/wiki/Development%20Environment

Reply:
Will do